gems/sorbet-runtime/Rakefile

```diff
@@ -11,8 +11,8 @@ task default: %i[test rubocop]
 def require_tests
   Dir
     .glob('./test/**/*.rb')
-    .reject {|path| path.match(%r{^./test/types/fixtures/})}
-    .reject {|path| path.match(%r{^./test/wholesome/})}
+    .reject {|path| path.match(%r{^./test/types/fixtures/}) }
+    .reject {|path| path.match(%r{^./test/wholesome/}) }
```

Comment:
Spaces after but not spaces before?!

Reply:
good catch, it didn't occur to me to change the next line in the .rubocop.yml file (it's wild that these are separately configurable)